microsoft-365/compliance/identify-a-hold-on-an-exchange-online-mailbox.md

```diff
@@ -261,6 +264,9 @@ The Hold Tracking Mailbox Diagnostics logs maintain a history of the holds appli
 $ht = Export-MailboxDiagnosticLogs <username> -ComponentName HoldTracking
 $ht.MailboxLog | Convertfrom-Json
 ```
+   
+   > [!NOTE]
+   > There will not be any hold tracking logs if the user account has been disabled. 
```

Comment:
```suggestion
> [!NOTE]
> There will not be any hold tracking logs if the user account has been disabled. 
```

Reply:
@JohanFreelancer9 The changes look great to me.